draft/draft-ietf-bfd-optimizing-authentication.xml

```diff
@@ -392,12 +395,51 @@
         excepting that Auth Type is still TBD and that Reserved is set to 2.
       </t>
 
-      <section title="Error Handling">
+      <section title="Receipt Using Optimized Authentication">
+        <t>
+          The procedures for authenticating BFD Control packets is similar to
+          the existing procedures covered in <xref target="RFC5880" section="6.7"/>. 
+          The Optimized Authentication mode provides requirements for
+          authentication key processing and how the result of the authentication
+          operation is stored in the authentication section for that mode.
+          Similarly, the value of the Auth Len field is specified by the
+          Optimized Authentication mode in each of their specifications.
```

Comment:
I am struggling with this paragraph, starting with the second sentence. Did you mean to say:

"The Optimized Authentication mode provides requirements for key steps in the authentication process and ..."?

Also, not clear on the last sentence. Would it be clearer to say:

"Similarly, the value of the Auth Len field is determined by the Optimized Authentication mode and is specified in the corresponding section."?

Reply:
Try my latest push to see if that helps clarify things.